.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,157 @@
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'The release_version used for the release branch name, e.g. release/vx.x.x'
+        default: 'vx.x.x'
+        required: true
+        type: string
+      base_branch:
+        description: "The base branch to release from; only override this if preparing a release off of the non-default branch"
+        default: 'main'
+        required: true
+        type: string
+
+env:
+  RELEASE_VERSION: ${{ inputs.release_version }}
+  RELEASE_BRANCH: release/${{ inputs.release_version }}
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-latest
```

Comment:
I'd been pinning this, but I guess for this job it's not super critical.

Reply:
Addressed in [5cafc476d335d536b7204ff747cca43372e3f93c](https://github.com/algorand/go-algorand-sdk/pull/737/commits/5cafc476d335d536b7204ff747cca43372e3f93c)